.projenrc.ts

```diff
@@ -53,8 +59,17 @@ const project = new cdk.JsiiProject({
       esModuleInterop: true,
     },
   },
+  gitignore: [
```

Comment:
Can you please elaborate why?

Reply:
dont know why that sneaked in - removing

Reply:
still there